index.html

```diff
@@ -221,22 +220,22 @@
         <h2><dfn>getEntries()</dfn> method</h2>
         <p>Returns a <a>PerformanceEntryList</a> object returned by
         <a href="#filter-buffer-by-name-and-type"></a> algorithm with
-        <var>buffer</var> set to <a>performance entry buffer</a>, and
+        <var>buffer</var> set to <a>relevant performance entry buffer</a>, and
```

Comment:
(same confusion with assuming single buffer) getEntries essentially looks at all the buffers and returns all of their entries

Reply:
So here, you'd probably want "relevant performance entry buffer for the context object" or some such.  Or you could define "relevant performance entry buffer", when used in an algorithm, to explicitly use the relevant global of https://dom.spec.whatwg.org/#context-object for that algorithm.

Reply:
@bzbarsky yep, that's what we have on line 202.

Reply:
Yep, that's exactly what we have in 202 and reference here.